tikv/tikv
# cloud: fix missing region config for aws

### What problem does this PR solve?

Problem Summary:
#9706 miss the region config for s3 storage. will cause error `The bucket you are attempting to access must be addressed using the specified endpoint.` This PR fix it.

### What is changed and how it works?

What's Changed:
set region config.

### Related changes

- Need to cherry-pick to the release branch after #9706 pick to release.

### Check List 

Tests 

- Unit test
- Manual test (add detailed scripts or steps below)
```
[root@CentOS76_VM s3_region]# AWS_ACCESS_KEY_ID=xxx AWS_SECRET_ACCESS_KEY=xxx./br restore table --db sysbench --table sbtest1 --pd 127.0.0.1:2379 -s s3://dbaas-benchmark/xxx/ --s3.region us-west-2 --check-requirements=false
Detail BR log in /tmp/br.log.2021-05-25T13.08.09+0800
Table restore <-----------------------------------------------------------/....................................................> 53.25%Table restore <---------------------------------------------------------------------------------------------------------------> 100.00%
Table restore <---------------------------------------------------------------------------------------------------------------> 100.00%
[2021/05/25 13:13:59.415 +08:00] [INFO] [collector.go:60] ["Table restore Success summary: total restore files: 39, total success: 39, total failed: 0, total take(Table restore time): 11m20.421168418s, total take(real time): 5m50.326250937s, total kv: 20000000, total size(MB): 2422.33, avg speed(MB/s): 3.56"] ["split region"=93.082593ms] ["restore checksum"=5m45.212433314s] ["restore ranges"=38] [Size=18356207286]
```

### Release note 

- Fix the issue that missing region config for aws s3.
